### PR TITLE
Publish the github pages as the helm repository

### PR DESCRIPTION
**Pull request by zhujian7**
issue reference: https://github.com/open-cluster-management-io/ocm/issues/486

### CODE REVIEW

File: charts/index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <title>Helm Charts Repository</title>
+</head>
+
+<body>
+    <h1>Open Cluster Management Helm Charts Repository</h1>
+    <p>This is the Helm charts repository for Open Cluster Management project. The index.yaml file is <a
+            href="index.yaml">here</a>.</p>
+</body>
+
+</html>
```
Comment (qiujian16) [inline charts/index.html]: EOF

File: charts/index.yaml [hunk 2]
```diff
@@ -0,0 +1,220 @@
+apiVersion: v1
```
Comment (qiujian16) [inline charts/index.yaml:1]: is this file auto-generated? what if we build a new version or new chart?

Comment (zhujian7) [reply charts/index.yaml:1]: this is generated by the command `helm repo index charts --url https://open-cluster-management.io/helm-charts/`, I think we should add a new workflow to auto-generate it.

Comment (zhujian7) [reply charts/index.yaml:1]: Let's merge this first to test integrating with artifactHub, then I will fix the comment with a follow-up PR.

Comment (qiujian16) [reply charts/index.yaml:1]: ok, this is a followup we should do , right?

**Pull request merged**